experiment/Commonfunctions/commonfunctions-exercise.js

```diff
@@ -182,29 +264,11 @@ function update(root) {
     .attr('y1',function(d){return d.source.y;})
     .attr('y2',function(d){return d.target.y;});
     d3.select("#nodes").moveToFront();
-    }
-
-
-
-
-function visitElement(element, animX) {
-    if(!tree_traversal.sequence_list[index])
-        return;
-    
-
-    d3.select("#node-" + element.id)
-        .transition().duration(tree_traversal.animDuration).delay(tree_traversal.animDuration * animX)
-        .style("fill", "rgb(158, 208, 62)").style("stroke", "rgb(158, 208, 62)");
-    setTimeout(function() {
-        d3.select("#traversal").append("text").text(element.name + " ");
-    }, 1.2 * tree_traversal.animDuration * animX);
-    document.getElementById("node-text" + element.id).style.pointerEvents = 'none';
-    tree_traversal.disabled_ids.push("node-text" + element.id);
 }
 
-
 function bft() {
     resetTraversal();
+    tree_traversal.sequence_list=[];
```

Comment:
It initially was like that but I started running into some errors since I had to manage DFT and BFT's exercises. Implementing this fixed those errors so I left it like that. Im not sure why it works like that but if you want me to delve deeper .